### DISCUSSION TITLE
Multiple github-pages sites stopped working

### DISCUSSION THREAD

**Discussion by psaikido** [community/community#127097] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a bunch of sites that have been up for ages but for some reason they now time out - "site took too long to respond". I've tried rebuilding, renewing the certificate by removing and re-saving, removing https altogether and adding it back in. How to troubleshoot this some more? 'dig hugle.uk' comes back as I'd expect. Removing the custom domain entirely gave me a psaikido.github.io/hugle.uk link which did the same thing, ie. timing out. This is a hugo site. There is another which is just raw html and that's doing the exact same thing over the last few days. Can anyone help me out here?

**Comment by LiteBrite82** [comment, marked as answer]
Hi @psaikido . 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

https://github.com/orgs/community/discussions/127077

The Community Discussions Team
